pypeit/coadd2d.py

```diff
@@ -943,17 +946,32 @@ def load_coadd2d_stacks(self, spec2d, chk_version=False):
             spat_flexure_list.append(s2dobj.sci_spat_flexure)
 
             sciimg_stack.append(s2dobj.sciimg)
+            exptime_stack.append(s2dobj.head0['EXPTIME'])
             waveimg_stack.append(s2dobj.waveimg)
             skymodel_stack.append(s2dobj.skymodel)
             sciivar_stack.append(s2dobj.ivarmodel)
             mask_stack.append(s2dobj.bpmmask.mask)
             slitmask_stack.append(s2dobj.slits.slit_img(flexure=s2dobj.sci_spat_flexure))
 
+        # check if exptime is consistent for all images
+        exptime_coadd = np.percentile(exptime_stack, 50., interpolation='higher')
+        if not np.all(exptime_stack == exptime_coadd):
```

Comment:
I think you want to use np.isclose here and not == since I'm not sure you want to rescale the exposure times if they are extremely close to each other. I guess it does not hurt, to do so anyway, but a user may get confused by this warning message.

Reply:
Using `np.isclose` with 1 second of tolerance. And not rescaling exposure times that are within 1 sec of difference. Does it sound OK?

Reply:
Yeah that is fine, but I don't see that change on line 958?

Reply:
@jhennawi here it is: https://github.com/pypeit/PypeIt/pull/1614/files#diff-75f38a683caf4fbc621639aba3c4e7c169b03be84035fee92f28dce72fbc031dR956